internal/api/identity.go

```diff
@@ -31,17 +33,28 @@ func (s *Server) CreateIdentity(ctx context.Context, request CreateIdentityReque
 		}, nil
 	}
 
+	var authBJJCredentialStatus verifiable.CredentialStatusType
+	if authBJJCredentialStatusString != nil && *authBJJCredentialStatusString != "" {
+		authBJJCredentialStatus = (verifiable.CredentialStatusType)(*authBJJCredentialStatusString)
```

Comment:
Any value different than "" can be accepted? Or we have a list of allowed values? In that case, we should check only for that allowed values. Otherwise is ok

Reply:
check for allowed values added